Makefile.release

```diff
@@ -86,6 +86,12 @@ tar:
 	    tar -zcf release/$(NAME)_$(VERSION)_linux_$$arch.tgz -C build/linux/$$arch $(NAME) ;\
 	done
 
+.PHONY: sum
+sum:
+	for asset in `ls -A release/*tgz`; do \
+	    sha256sum $$asset > $$asset.sha256; \
```

Comment:
@miekg Why do you publish a separate sha256sum file for each asset? I think the idiomatic way is to provide a single file (often called `sha256sum.txt`, or `SHA256SUMS`) with one line per artifact. Providing separate files seems to just unnecessarily complicate the process of redistributing binaries while ensuring consistency.